ashley/src/com/badlogic/ashley/core/Engine.java

```diff
@@ -60,6 +63,20 @@ public Entity createEntity () {
 	}
 
 	/**
+	 * Creates a new {@link Component}. To use that method your components must have a visible no-arg constructor
+	 */
+	public <T extends Component> T createComponent (Class<T> componentType) {
```

Comment:
This is using the regular Java reflection API, you need to use the [Libgdx reflection API](https://github.com/libgdx/libgdx/wiki/Reflection), otherwise you will break the GWT backend.

I already did [this](https://github.com/libgdx/ashley/blob/serialization/ashley/src/com/badlogic/ashley/core/Engine.java#L59) in the serialization branch, you can just copy it.